.github/workflows/test.yml

```diff
@@ -210,12 +210,13 @@ jobs:
       #     nats-server.exe -v
       #     dotMemoryUnit $env:userprofile\.dotnet\tools\nunit.exe --propagate-exit-code -- .\tests\NATS.Client.Core.MemoryTests\bin\Release\net6.0\NATS.Client.Core.MemoryTests.dll
 
+      # fails when target frameworks are run in parallel; run tests with -p:TestTfmsInParallel=false
```

Comment:
Not entirely sure why this started happening with xunit v3, but this workaround appears to fix